packages/edit-site/src/components/sidebar-edit-mode/settings-header/index.js

```diff
@@ -35,7 +30,9 @@ const SettingsHeader = ( { sidebarName } ) => {
 		};
 	} );
 
-	const entityLabel = entityLabels[ entityType ] || entityLabels.wp_template;
+	const entityLabel =
+		POST_TYPE_LABELS[ entityType ] ||
+		POST_TYPE_LABELS[ TEMPLATE_POST_TYPE ];
```

Comment:
There is a very subtle change here — we can now match against `TEMPLATE_PART_POST_TYPE` to get the `Template part` label, which wasn't possible before. This seems like a good change to me 👍

| Trunk | This PR |
| --- | --- |
| <img width="286" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/1c553cab-d66a-467d-b309-55b7dbdfb918"> | <img width="284" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/8dde6933-3bc1-43cc-8f03-fec56ffc64f9"> |

I noticed one of the Playwright tests was failing. I've just kicked it off again, but if it's still failing, perhaps due to this label being updated?

This is the line where I think it might need to be updated to `Template part`?

https://github.com/WordPress/gutenberg/blob/420013b3f3e24f00a4dcfb80d4a173dacca92849/test/e2e/specs/site-editor/writing-flow.spec.js#L69

Reply:
Thanks for tracking it down! I just pushed and walked away :D 

It makes me realize how many variations of Template part, template part and Template Part we have across the codebase.

I've updated to match the constant for now (Template Part). This was what I gleaned from https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/#capitalizing-wordpress-terms.

I can make a note to unify them once we decide on what it should be 😄

Reply:
> It makes me realize how many variations of Template part, template part and Template Part we have across the codebase.

Oh, good point! At least this e2e test is case insensitive so it shouldn't break when we unify them further down the track 🙂

Reply:
Related conversation in https://github.com/WordPress/gutenberg/issues/51744